subscription-manager.spec

```diff
@@ -919,8 +1008,8 @@ find %{buildroot} -name \*.py -exec touch -r %{SOURCE0} '{}' \;
 %endif
 %{_bindir}/rhsm-icon
 
-%{_datadir}/gnome/help/subscription-manager/C/figures/*.png
-%{_datadir}/gnome/help/subscription-manager/C/*.xml
+%doc %{_datadir}/gnome/help/subscription-manager/C/figures/*.png
+%doc %{_datadir}/gnome/help/subscription-manager/C/*.xml
```

Comment:
Why are you marking these as doc files?

Reply:
Because the OBS says so to prevent a rpmlint warning. I can revert if you guys like.

Reply:
@Conan-Kudo what do you think?

Reply:
My concern would be if the subman GUI tolerates the help data not being present. Some applications do, and some don't...

Reply:
What exactly changes with the %doc macro?
Isn't this some rpm internals. This shouldn't affect the subman GUI in any way, as I didn't seperate content into a new package

Reply:
The doc macro makes it so you can filter out the files from being installed by passing `nodocs` as a transaction filter.

Reply:
ah thank you very much. Then I revert it